src/api/apps/graphql/resolvers.js

```diff
@@ -271,7 +271,6 @@ export const seriesArticle = root => {
   return new Promise(async (resolve, reject) => {
     const seriesArticles = await promisedMongoFetch({
       layout: "series",
-      published: true,
```

Comment:
Help me understand what's happening here - we had initially been saying that `published` was automatically true, but that was a bug? Or this is being removed and appended elsewhere, but I'm not seeing it (it only shows up in the integration mocks?)

Is this change being captured in any tests anywhere? Should we add a third mock to the integration test that isn't published and ensure that it comes through that way, or?

Reply:
I think this is b/c we need to return both published and unpublished articles so the bool check is removed. We need to fetch unpublished articles b/c editorial is adding content to the series every day, as such we need to be able to fetch the articles that are still in draft mode in order to render them at the `/series/some-vanguard-slug` route. But @eessex please correct me if I'm wrong!

Reply:
The articles api does not return unpublished content from users who do not have channel access, so this is extra -- it is hiding unpublished content from admins, making them unable to properly spec drafts.